WalletWasabi/Stores/SqliteStorage.cs

```diff
@@ -0,0 +1,241 @@
+using Microsoft.Data.Sqlite;
+using NBitcoin;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using WalletWasabi.Backend.Models;
+
+namespace WalletWasabi.Stores;
+
+/// <summary>
+/// Sqlite-based storage for block filters.
+/// </summary>
+/// <seealso href="https://learn.microsoft.com/en-gb/dotnet/standard/data/sqlite/async">
+/// Async is not supported at the moment, so no async methods were used in this implementation.
+/// </seealso>
+public class SqliteStorage : IDisposable
```

Comment:
There's no derive classes expected, so it can be made sealed. With moving to short living connections, `IDisposable` isn't needed.